actix/src/sec-6-sync-arbiter.md

```diff
@@ -1 +1,66 @@
-[WIP] 
\ No newline at end of file
+# SyncArbiter
+
+When you normally run Actors, there are multiple Actors sharing a single
+thread and event loop in the System. However for CPU bound workloads, or
+highly concurrent workloads, you may wish to have a single Actor able
+to run on a pool of OS threads.
+
+This is what a SyncArbiter provides - an enviroment allowing a single Actor
+to be run on multiple threads concurrently.
+
+It's important to note a SyncArbiter can only host a single type of Actor. If
+you want to have two types of Actors running, you'll need to define two SyncArbiters
+
+## Creating a Sync Actor
+
+When constructing your Actor, you have to change `Context` to `SyncContext`
```

Comment:
`When constructing your Actor,`

When implementing Actor?

`you have to change `Context` to `SyncContext`

It doesn't define that actor will run on `SyncArbiter`, instead it is `SyncArbiter` who requires that your Actor's Context type is `SyncContext`

Reply:
Changed